### PR TITLE
3.2.0 Release

### PR DESCRIPTION
**Pull request by phenaproxima**
This is pretty much ready to go. To roll the release from here:

* `composer update`
* Make the 3.2.0 fixture.
* Tag and push.
* Release on drupal.org with release notes.
* Release on GitHub with tarball (`./tarball.sh 3.200`).
* Back to dev.

That should be everything needed to tag and release, once CI is green.

### CODE REVIEW

File: UPDATE.md [hunk 1]
```diff
@@ -170,6 +170,9 @@ are currently running 2.2.0 and are trying to update to 2.2.6, you will need to
 follow the instructions for updating from 2.2.0 to 2.2.1, then from 2.2.1 to
 2.2.2, in that order.
 
+### 3.2.0-beta1 to 3.2.0
+There are no manual update steps for this version.
```
Comment (balsama) [inline UPDATE.md]: What about the sub-profile API change?

Comment (phenaproxima) [reply UPDATE.md]: That is explained further down in the file (I think under beta1).

Comment (balsama) [reply UPDATE.md]: Oh. Looks like we can combine the 3.2.0-beta1 update steps?

Comment (phenaproxima) [reply UPDATE.md]: Discussed in Slack, agreed to remove beta1. Done.

**Pull request merged**